.eslintignore

```diff
@@ -17,3 +17,10 @@ compiled.d.ts
 playwright.config.js
 
 ts/localization/locales.ts
+
+about_preload.js
+commitlint.config.js
+libsession.worker.config.js
+password_preload.js
+sass.config.js
+utils.worker.config.js
```

Comment:
Why ignore the workers?

Reply:
those are the gerenated .js files, not the .ts